torchgeo/datamodules/so2sat.py

```diff
@@ -14,79 +16,232 @@
+                0.15428468872076,
+                0.10905050699570,
+            ]
+        ),
+    }
+
+    stds_per_version: dict[str, Tensor] = {
+        "2": torch.tensor(
+            [
+                0.17555201,
+                0.17556463,
+                0.45998793,
+                0.45598876,
+                2.85599092,
+                8.32480061,
+                2.44987574,
+                1.46473530,
+                0.03958795,
+                0.04777826,
+                0.06636616,
+                0.06358874,
+                0.07744387,
+                0.09101635,
+                0.09218466,
+                0.10164581,
+                0.09991773,
+                0.08780632,
+            ]
+        ),
+        "3_random": torch.tensor(
+            [
+                0.1756914,
+                0.1761190,
+                0.4600589,
+                0.4563601,
+                2.2492179,
+                7.9056503,
+                2.1917633,
+                1.3148480,
+                0.0392269,
+                0.0470917,
+                0.0653264,
+                0.0624057,
+                0.0758367,
+                0.0891717,
+                0.0905092,
+                0.0996856,
+                0.0990188,
+                0.0873386,
+            ]
+        ),
+        "3_block": torch.tensor(
+            [
+                0.1751797,
+                0.1754073,
+                0.4610124,
+                0.4572122,
+                0.8294254,
+                7.1771026,
+                0.9642598,
+                0.8770835,
+                0.0388311,
+                0.0464986,
+                0.0643833,
+                0.0616141,
+                0.0753004,
+                0.0886178,
+                0.0899500,
+                0.0991759,
+                0.0983276,
+                0.0865943,
+            ]
+        ),
+        "3_culture_10": torch.tensor(  # note: this is the same as "2"
+            [
+                0.17555201,
+                0.17556463,
+                0.45998793,
+                0.45598876,
+                2.85599092,
+                8.32480061,
+                2.44987574,
+                1.46473530,
+                0.03958795,
+                0.04777826,
+                0.06636616,
+                0.06358874,
+                0.07744387,
+                0.09101635,
+                0.09218466,
+                0.10164581,
+                0.09991773,
+                0.08780632,
+            ]
+        ),
+    }
 
     def __init__(
         self,
         batch_size: int = 64,
         num_workers: int = 0,
+        version: str = "2",
         band_set: str = "all",
+        validation_pct: float = 0.2,
         **kwargs: Any,
     ) -> None:
         """Initialize a new So2SatDataModule instance.
 
         Args:
             batch_size: Size of each mini-batch.
+            version: One of "2" or "3_random", "3_block", or "3_culture_10".
```

Comment:
We don't need a `version` parameter, we can pass it directly through `**kwargs` to the dataset. See bigearthnet/oscd/seco for an example where we pull the info we need from kwargs and use it to set mean/std dev.